src/components/events/list-view/EventCard.tsx

```diff
@@ -9,15 +9,26 @@ interface EventCardProps {
   currentPage: number;
   filter?: string;
 }
+function checkNew(createdAt: Date) {
+  const currentDate = new Date();
+  const millisecondsIn3Days = 3 * 24 * 60 * 60 * 1000;
+  const timeDifference = Math.abs(createdAt.getTime() - currentDate.getTime());
+  if (timeDifference <= millisecondsIn3Days) {
+    return "float-right m-5";
+  }
+  return "hidden";
+
+}
 
 function EventCard({ event, currentPage, filter }: EventCardProps) {
   return (
     <Card className="hover:bg-gray-200 dark:hover:bg-card/50 transition-colors duration-300">
       <Link
         key={event.id}
         href={`/events/${event.id}?page=${currentPage}${filter ? `&filter=${filter}` : ""}`}>
+        <img className={checkNew(event.createdAt)} width="80" height="80" src="/New.png" alt="" />
```

Comment:
Optimization 2: The image is currently 10.3 KB. I would recommend considering the use of an absolutely positioned element with text instead of displaying an image.